packages/front-end/components/HealthTab/DimensionIssues.tsx

```diff
@@ -0,0 +1,287 @@
+                    <div className="mt-4">
+                      <div className="mb-2">
+                        <VariationUsersTable
+                          users={d.variationUnits}
+                          variations={variations}
+                          srm={d.srm}
+                        />
+                        {(d.health === "healthy" ||
+                          d.health === "Issues detected") && (
+                          <SRMWarning
+                            srm={d.srm}
+                            variations={variations}
+                            users={d.variationUnits}
+                            showWhenHealthy
+                            type="simple"
+                          />
+                        )}
+                        {d.health === "Not enough traffic" && (
+                          <div className="alert alert-info">
+                            <b>
+                              More traffic is required to detect a Sample Ratio
+                              Mismatch (SRM).
+                            </b>
+                          </div>
+                        )}
+                      </div>
+                    </div>
+                  </HealthCard>
+                );
+              })}
+            </>
+          )}
+        </div>
+      </Modal>
+
+      <div className="d-flex flex-column h-100">
+        <div className="px-4">
+          <h3>Dimensions</h3>
+          <p className="mt-1">Highlights perceived issues across dimensions</p>
+        </div>
+
+        <hr className="my-0 w-100" />
+        {areDimensionsAvailable ? (
+          <>
+            <div
+              className="flex-fill flex-shrink-1 overflow-auto px-4"
+              style={{ paddingTop: "12px" }}
+            >
+              {availableDimensions.map((d) => {
+                return (
+                  <div key={d.value}>
+                    <a
+                      href="#"
+                      onClick={(e) => {
+                        e.preventDefault();
+                        setSelectedDimension(d.value);
+                        track("Open health tab dimension modal");
+                        setModalOpen(true);
+                      }}
+                    >
+                      <p style={{ marginBottom: "2px" }}>
+                        <b>{d.label}</b>
+                      </p>
+                    </a>
+                    <p>
+                      {d.issues.length ? (
+                        <>
+                          <b>Issues: </b>
+                          {d.issues.join(", ")}
+                        </>
+                      ) : (
+                        <i className="text-muted">No issues detected</i>
+                      )}
+                    </p>
+                  </div>
+                );
+              })}
+            </div>
+            {/*TODO: if size of dimension list area is greater than header + footer add boxShadow. Hide otherwise.*/}
```

Comment:
i thought we solved this one with ref and resize listeners

Reply:
oh whoops I meant to delete this

Reply:
Oh wait no I used the ref to make sure the card container was sized according to the left side of the card with the table. The resize listener was used to make sure that the container was resized when the window was resized. I can probably do the same approach here though. I'll try to get that working real quick but if I run out of time I'll just do it as a fast follow